corehq/apps/domain/project_access/middleware.py

```diff
@@ -3,13 +3,17 @@
 from django.utils.deprecation import MiddlewareMixin
 from corehq.apps.domain.project_access.models import SuperuserProjectEntryRecord, ENTRY_RECORD_FREQUENCY
 from corehq.util.quickcache import quickcache
+from corehq.apps.users.tasks import update_domain_date
 
 
 class ProjectAccessMiddleware(MiddlewareMixin):
     def process_view(self, request, view_func, view_args, view_kwargs):
         if getattr(request, 'couch_user', None) and request.couch_user.is_superuser \
                 and hasattr(request, 'domain'):
             return self.record_entry(request.domain, request.couch_user.username)
+        if getattr(request, 'couch_user', None) and request.couch_user.is_web_user() \
+                and hasattr(request, 'domain'):
+            update_domain_date(request.couch_user, request.domain)
```

Comment:
This calls the task synchronously, is that on purpose? If so, not sure it needs to be a celery task

Reply:
No, I just forgot to add a `.delay` because I do want this to run asynchronously. Thanks for catching that!